tests/prowjobsscraper/test_step.py

```diff
@@ -5,8 +5,8 @@
 import pkg_resources
 from google.cloud import exceptions
 
-from prowjobsscraper import step
-from prowjobsscraper.prowjob import ProwJobs
+from projects.prowjobsscraper import step
```

Comment:
I think it's kind of a convention that repository name and the root package have the same name, ``projects`` kind of negates this concept (I would expect ``prowjobsscraper`` to be the root, or maybe we can rename the repo name to something more generic)

Reply:
Yes I agree, I will change the repository name as well, do you have an idea ?

Reply:
The best I can think right now is ``prow-jobs-reporter``

Reply:
I named it `prow-insights` in app-interface, we could align with this name?